Pages/ChromeFirstRun.html

```diff
@@ -258,163 +297,313 @@ <h1 class="page-header">
+            Blog Post Regarding Release
+           </a>
+          </li>
+          <li>
+           Updated the first-run document.
+          </li>
+         </ul>
+         <p>
+          <strong>
+           Version 0.3.0
+          </strong>
+         </p>
+         <ul>
+          <li>
+           <a href="http://www.privly.org/content/chrome-extension-alpha-release">
+            Blog Post Regarding Release
+           </a>
+          </li>
+          <li>
+           Added the first-run document.
+          </li>
+          <li>
+           Updated page styling.
+          </li>
+         </ul>
+         <p>
+          <strong>
+           Version 0.2.0
+          </strong>
+         </p>
+         <ul>
+          <li>
+           Added content server selection.
+          </li>
+          <li>
+           Added user-defined whitelists.
+          </li>
+          <li>
+           Added options page.
+          </li>
+          <li>
+           Added toggling capabilities.
+          </li>
+          <li>
+           Added loading message.
+          </li>
+          <li>
+           Added anti-spoofing glyph.
+          </li>
+          <li>
+           Added local PlainPosts.
+          </li>
+          <li>
+           Added local
+           <a href="https://github.com/privly/privly-chrome/issues/15" target="_blank">
+            ZeroBins
+           </a>
+           .
+          </li>
+          <li>
+           Adopted Google's Native Client for compiled library architecture.
+          </li>
+         </ul>
+         <p>
+          <strong>
+           Version 0.1.1
+          </strong>
+         </p>
+         <ul>
+          <li>
+           Posting directly to content servers
+          </li>
+          <li>
+           Context menu for posting ZeroBins
+          </li>
+          <li>
+           Context menu for posting PlainPosts
+          </li>
+          <li>
+           Modal button for turning the extension off
+          </li>
+         </ul>
+         <h2>
+          Privly Resources
+         </h2>
         </div>
        </div>
       </div>
-      <!-- /container -->
      </div>
     </div>
    </div>
-   <!-- /.container -->
-   <div class="container">
-    <div class="row">
-    </div>
-    <!-- /.row -->
-   </div>
-   <!-- /.container -->
-  </div>
-  <!-- /.wrap -->
-  <div id="footer">
-   <div class="container">
-    <p class="text-muted credit">
-     You are viewing a Privly
-     <span style="vertical-align: super;font-size: xx-small">
-      ®
-     </span>
-     Application,
-     <a href="http://priv.ly" target="_blank">
-      learn more
-     </a>
-     about Privly.
-     <a class="legal_nav" href="http://priv.ly" target="_blank">
-      Terms of Service
-     </a>
-     and
-     <a class="legal_nav" href="http://priv.ly" target="_blank">
-      DMCA
-     </a>
-     .
-    </p>
-   </div>
   </div>
  </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
The templating system will resolve an unclosed tag by putting nodes *after* closing the `<html>` tag. You can usually resolve it by deleting content until the build fixes itself, then look in the deleted content for the problem.